### PR TITLE
Return addresses in PhoenixSplit.getInfo()

### LINKED ISSUE #10757: Add more information about the split in PhoenixSplit getInfo()
In the Phoenix Connector logs - we see below which doesn't give much information about the Phoenix Split. Looking at other connectors, it looks to make things better to add more information here like the regionServer location etc to make it easy to discover phoenix server side things.

DEBUG	<TaskId>	io.trino.execution.executor.TaskExecutor	Split <TaskId>-3 io.trino.plugin.phoenix.**PhoenixSplit@48b4b96a**

### PR DESCRIPTION
**Pull request by rameshbabu-sfdc**
## Description

> Is this change a fix, improvement, new feature, refactoring, or other?
logging change
> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)
connector
> How would you describe this change to a non-technical end user or system administrator?
logging addresses for debug purpose
## Related issues, pull requests, and links

* Fixes #10757

## Documentation

( x) No documentation is needed.
( ) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

( ) No release notes entries required.
( ) Release notes entries required with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: plugin/trino-phoenix/src/main/java/io/trino/plugin/phoenix/PhoenixSplit.java [hunk 1]
```diff
@@ -74,4 +74,11 @@ public long getRetainedSizeInBytes()
                 + estimatedSizeOf(addresses, HostAddress::getRetainedSizeInBytes)
                 + serializedPhoenixInputSplit.getRetainedSizeInBytes();
     }
+
+    @Override
+    public Object getInfo()
+    {
+        //return the addresses
+        return getAddresses();
```
[the remark this answers is not in the document]
Comment (rameshbabu-sfdc) [reply plugin/trino-phoenix/src/main/java/io/trino/plugin/phoenix/PhoenixSplit.java]: done

**Review by rameshbabu-sfdc — COMMENTED** [review]
Implemented the required suggestions @ebyhr

File: plugin/trino-phoenix/src/main/java/io/trino/plugin/phoenix/PhoenixSplit.java [hunk 2]
```diff
@@ -74,4 +75,13 @@ public long getRetainedSizeInBytes()
...
     }
+
+    @Override
+    public Object getInfo()
+    {
+        return ImmutableMap.builder()
+                .put("addresses", addresses)
+                .buildOrThrow();
+    }
+
```
Comment (ebyhr) [inline plugin/trino-phoenix/src/main/java/io/trino/plugin/phoenix/PhoenixSplit.java:86]: Remove an empty line.

**Review by ebyhr — APPROVED** [review]
Reminder
> Also, please squash commits into one and fix the commit title (≠ PR title) as "Return addresses in PhoenixSplit.getInfo()".
https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages

**Comment by rameshbabu-sfdc** [comment]
@ebyhr    docker tests failed with the below error. Any idea how to resolve this ?

Warning: rning] The requested image's platform (linux/arm64) does not match the detected host platform (linux/amd64) and no specific platform was requested

**Pull request merged**